rhel/rhel-8.x/rhel-8.6-hpc/install_intel_libs.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$ALMA_COMMON_DIR/install_intel_libs.sh
```

Comment:
I see you copied `install_intel_libs.sh` in rhel/common, but if I understand correctly this line means that it will use the version from alma?